docs/cookbooks/multus/README.md

```diff
@@ -0,0 +1,328 @@
+# Using Antrea with Multus
+
+This guide will describe how to use Project Antrea with
+[Multus](https://github.com/intel/multus-cni), in order to attach multiple
+network interfaces to Pods. In this scenario, Antrea is used for the default
+network, i.e. it is the CNI plugin which provisions the "primary" network
+interface ("eth0") for each Pod. For the sake of this guide, we will use the
+[macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan)
+CNI plugin to provision secondary network interfaces for selected Pods, but
+similar steps should apply for other plugins as well,
+e.g. [ipvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/ipvlan).
+
+## Prerequisites
+
+The only prerequisites are:
+ * a K8s cluster (Linux Nodes) running a K8s version supported by Antrea. At the
+   time of writing, we recommend version 1.16 or later. Typically the cluster
+   needs to be running on a network infrastructure that you control. For
+   example, using macvlan networking will not work on public clouds like AWS.
+ * [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+All the required software will be deployed using YAML manifests, and the
+corresponding container images will be downloaded from public registries.
+
+macvlan requires the network to be able to handle "promiscuous mode", as the
```

Comment:
I feel this description is not accurate, for promisc mode is configured on the NIC device, but not on the network, and if the underlay network supports MAC learning promisc mode NICs should work fine.
But I know some hypervisor networking requires promisc mode settings on the vSwitch port, and other documentation describes it the same way, so probably not a big deal.

Reply:
And this requirement should be for macvlan bridge mode only?

Reply:
I clarified by mentioning that this is for macvlan "bridge" mode.

Regarding your first comment, I don't think the following is too misleading:
> macvlan requires the network to be able to handle "promiscuous mode"

A lot of pieces can be involved:
* hypervisor
* physical NIC
* underlying network: won't work on public cloud (as you said it needs to support MAC learning)

Reply:
Not disagree what you said.